detox/src/android/espressoapi/DetoxAction.js

```diff
@@ -271,6 +271,92 @@ class DetoxAction {
     };
   }
 
+  static longPress() {
```

Comment:
Oh... wow.. is this generated by `generation` project? If not, it won't work as we overwrite this file automatically on each build.

Reply:
it is